src/encoded/schemas/file_format.json

```diff
@@ -127,22 +127,106 @@
                 "description": "Item class name (e.g. FileFastq)",
                 "type": "string"
             }
+        },
+        "assay_types": {
+            "title": "Assay types",
+            "description": "Assays that make use of this file format",
+            "type": "array",
+            "items": {
+                "title": "Assay or Category",
+                "type": "string",
+                "suggested_enum": [
+                    "Sequencing",
+                    "Microscopy",
+                    "Hi-C",
+                    "SPRITE",
+                    "RNA-seq",
+                    "Generic"
```

Comment:
Is this a catchall?  What format(s) might include generic and how would that fit into the result table?

I realize that this is a suggested enum and I know you've been thinking about this - I just don't have a good sense as to if you think these values currently will be sufficient or is the plan to add more as you work on the look of the embedded search table?

Reply:
As far as “Generic” in the file_formats assay type, this is meant for formats like “txt” meaning that it isn’t for a particular assay type but is a generic file format. The other option would be to remove that from the enum and leave that field blank for those file formats.

As to whether the values in the enum are sufficient, i won’t be surprised if changes are needed later on. Maybe replacing “SPRITE” with “multi-contact assays” or something could make sense. For our current purposes though I felt the values were sufficient - 1D sequencing experiments use formats like bed and bw and I think “Sequencing” is an appropriate category for that, without specifying what kind of sequencing.